tutor/src/screens/gradebook/student-data-sorter.js

```diff
@@ -2,10 +2,12 @@ import { isNumber } from 'lodash';
 
 const percent = (num, total) => Math.round((num / total) * 100) || 0;
 
-const getSortValue = function(student, index, dataType, displayAs) {
-
-  if (!isNumber(index)) { return (student.last_name || student.name).toLowerCase(); }
-  const task = student.data[index];
+const getSortValue = function(student, key, dataType, displayAs) {
+  //name, course_average, homework_score, reading_score
```

Comment:
😁 left in by mistake I presume?

Reply:
oh no, I just added this to know what the keys might be set to. I should make this comment more readable.